src/shared/telemetry/activation.ts

```diff
@@ -54,13 +54,17 @@ export async function activate(activateArguments: {
     // When there are configuration changes, update the telemetry service appropriately
     vscode.workspace.onDidChangeConfiguration(
         async event => {
-            if (!ext.telemetry) {
-                return
-            }
             if (
                 event.affectsConfiguration('telemetry.enableTelemetry') ||
                 event.affectsConfiguration('aws.telemetry')
             ) {
+                if (!ext.telemetry) {
+                    getLogger().warn(
+                        'Telemetry configuration changed, but telemetry is undefined. This can happen during testing.'
```

Comment:
mentioning https://github.com/aws/aws-toolkit-vscode/issues/1071 or just `#1071` in the message can save time
```suggestion
                        'Telemetry configuration changed, but telemetry is undefined. This can happen during testing. #1071'
```